src/pycsou/abc/operator.py

```diff
@@ -2016,6 +2016,64 @@ def dagger(self, damp: typ.Optional[float] = None, **kwargs) -> "LinOp":
         )
         return dagger
 
+    @classmethod
+    def from_sciop(cls, scipy_operator) -> "LinOp":
+        r"""
+        Cast a :py:class:`scipy.sparse.linalg.LinearOperator` instance into a :py:class:`~pycsou.abc.operator.LinOp`
+        instance, compatible with the matrix-free linear algebra routines of `scipy.sparse.linalg <https://docs.scipy.org/doc/scipy/reference/sparse.linalg.html>`_.
+
+        Parameters
+        ----------
+        scipy_operator:  scipy.sparse.linalg.LinearOperator | cupyx.scipy.sparse.linalg.LinearOperator
+            Linear operator object compliant with SciPy's interface.
+
+        Returns
+        -------
+        pycsou.abc.operator.LinOp
+           Output is a :py:class:`~pycsou.abc.operator.LinOp` object.
+
+        Examples
+        --------
+        >>> import numpy as np
+        >>> from pycsou.abc.operator import LinOp
+        >>> from scipy.sparse.linalg import aslinearoperator
+        >>> mat = np.array([[1, 2, 3], [4, 5, 6]], dtype=np.int32)
+        >>> sciop = aslinearoperator(mat)
+        >>> linop = LinOp.from_sciop(sciop)
+        >>> x = np.arange(15).reshape(5, 3)
+        >>> np.allclose(linop(x), sciop.matmat(x.T).T) # transpose to satisfy numpy n-dim convention
+        True
+
+
+        .. Warning::
+        This  is a simplified example for illustration puposes only. It may not abide by all the rules listed in the
+        :ref:`developer-notes`.
+
+        See Also
+        --------
+        :py: meth:`~pycsou.abc.operator.LinOp.from_array`, :py: meth:`~pycsou.abc.operator.Map.from_source`
```

Comment:
Careful you have an unecessary space in both your links. It should be: ``:py:meth:`~pycsou.abc.operator.LinOp....``
Also add a link to the method ``LinOp.to_sciop``.